_posts/2018-07-11-cockpit-172.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Cockpit 172
+author: pitti
+date: 2018-07-11
+tags: cockpit linux
+slug: cockpit-172
+category: release
+summary: Cockpit with on-demand PCP install, flow control, and Python 3 support
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 172.
+
+### System: Offer installation of PCP
+
+The System page now shows an "Enable persistent metrics…" link if the
+[PCP](https://pcp.io/) package is not installed. This is similar to the
+on-demand installation of the NFS client packages in
+[Cockpit 169](https://cockpit-project.org/blog/cockpit-169.html).
+
+![Install PCP on demand](/images/pcp-install-on-demand.png)
+
+### Software Updates: Improve layout in mobile mode
+
+The Software Updates page fixed some incorrect spacing and layout in small
+browser windows and mobile browsers:
+
+![PackageKit mobile mode](/images/packagekit-mobile-optimization.png)
+
+
+### Remove ability to drop privileges from navigation bar
+
+Cockpit previously showed a "Locked" or "Unlocked" status in the navigation
+bar, reflecting whether or not the "Reuse my password for privileged tasks"
+check box was enabled at log in. Clicking on "Unlocked" would lock the
+interface and drop administrator capabilities.
```

Comment:
Before this release, Cockpit showed a “Locked” or “Unlocked” status in the navigation bar. It reflected the “Reuse my password for privileged tasks” checkbox on the login page. Clicking on “Unlocked” would lock the interface and drop administrator capabilities.